api/tests/unit/application/certification-point-of-contacts/certification-point-of-contact-controller_test.js

```diff
@@ -45,6 +45,7 @@ describe('Unit | Controller | certifications-point-of-contact-controller', funct
             'first-name': 'Buffy',
             'last-name': 'Summers',
             email: 'buffy.summers@example.net',
+            lang: 'fr',
```

Comment:
Remarque : on teste le serializer aussi en faisant ça car il n'est pas stubbé dans ce TU.
Cf ce qu'on fait dans `Unit | Controller | admin-member-controller`
Qu'en penses tu ? On soumet à l'équipe pour savoir si on dit que c'est un standard ?

Reply:
Je pense que je comprends. J'ai bien noté d'aborder ce point au prochain atelier avec l'équipe.